source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1282,7 +1282,15 @@ export abstract class DesignerPeer extends DraggableElement {
 }
 
 export class ActionPeer extends DesignerPeer {
-    static readonly titleProperty = new StringPropertyEditor(Adaptive.Versions.v1_0, "title", "Title");
+    static readonly titleProperty = new StringPropertyEditor(Adaptive.Versions.v1_0, "title", "Title", true);
+    static readonly modeProperty = new ChoicePropertyEditor(
+        Adaptive.Versions.v1_5,
+        "mode",
+        "Mode",
+        [
+            { targetVersion: Adaptive.Versions.v1_5, name: "Primary", value: Adaptive.ActionMode.Primary },
+            { targetVersion: Adaptive.Versions.v1_5, name: "Secondary", value: Adaptive.ActionMode.Secondary }
+        ]);
```

Comment:
did you intend to have the mode property in this PR, or is the feature branch out of date?

Reply:
See, this is what I've been talking about. The plan for JS was that I was going to push changes directly into js-1.5-preview, not in those feature branches. This is getting complicated already. So now I have two choices:
1. Either I remove those mode-related changes from this PR
2. Or I retarget this PR back to js-1.5-preview

My choice is very much option #2.

Reply:
To be clear, this mode-related work is just something I implemented opportunistically in the same branch as the heading (and tooltip) stuff because it was all supposed to be merged directly into js-1.5-preview. The mode work has a dependency on code that's currently only in js-1.5-preview, namely the overflow menu feature.